WordPress/Jetpack/AppConfiguration.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 
-@objc class AppConfiguration: NSObject, TargetConfiguration {
+@objc class AppConfiguration: NSObject {
     @objc static let isJetpack: Bool = true
+    @objc static let allowSiteCreation: Bool = true
```

Comment:
Should this be false? 

```suggestion
    @objc static let allowSiteCreation: Bool = false
```

Reply:
Uh oh.